junit-testcontainer-rabbitmq/src/main/java/io/appform/testcontainers/rabbitmq/container/RabbitMQContainer.java

```diff
@@ -27,7 +28,8 @@ public RabbitMQContainer(final RabbitMQContainerConfiguration rabbitMQContainerC
             .withExposedPorts(rabbitMQContainerConfiguration.getPort(), rabbitMQContainerConfiguration.getManagementPort())
             .withLogConsumer(ContainerUtils.containerLogsConsumer(log))
             .waitingFor(new RabbitMQStatusCheck(rabbitMQContainerConfiguration))
-            .withStartupTimeout(rabbitMQContainerConfiguration.getTimeoutDuration());
+            .withStartupTimeout(rabbitMQContainerConfiguration.getTimeoutDuration())
+            .withStartupCheckStrategy(new IsRunningStartupCheckStrategyWithDelay());
```

Comment:
This basically fixes a startup delay of 1 second blindly. Make this configurable as a constructor actually exists for doing this. `getTimeoutDuration` should ideally be deprecated or ignored if this is used or use the timeout to configure the retrier (see above comment) in the strategy..